docs/examples/all-ligand-explicit.rst

```diff
@@ -0,0 +1,184 @@
+.. _all-ligand-explicit:
+
+Absolute Binding of Binders and Non-binders to T4 lysozyme L99A in Explicit Solvent with Combinatorial Options
+==============================================================================================================
+
+This example looks at the absolute binding of a series of binders and non-binder to the T4 Lysozyme L99A protein.
+
+We take advantage of three features of YANK in this example we have not done in others thus far:
```

Comment:
Let's not assume anyone will work through tutorials serially! Everyone will just dive in directly to the tutorial that is closest to the one they are interested in. If you reference a simpler tutorial, link to it and provide a simple description.